.ghal.rules.json

```diff
@@ -29,6 +30,9 @@
               "(?i)dotnet-roslyn-api$": ":books: Area - Roslyn API Reference,:file_folder: Repo - roslyn-api-docs",
               "(?i)dotnet-architecture$": ":books: Area - .NET Architecture Guide"
             },
+            "technology": {
+              "(?i)dotnet-wpf$": ":card_file_box: Technology - WPF"
```

Comment:
I'm looking at the approved metadata values and this is not supported

Reply:
@mairaw 
I filed a ticket yesterday.. I didn't realize that I needed to associate the technology with a service. The service was approved, I don't see why the tech wouldn't be also.